src/components/conference-card.tsx

```diff
@@ -0,0 +1,291 @@
+}
+
+
+type TalkStateProps = {
+  id: string;
+  collection: string;
+  body: string;
+  slug: string;
+  data: Talk;
+}
+
+type SpeakerStateProps = {
+  id: string;
+  collection: string;
+  body: string;
+  slug: string;
+  data: Speaker;
+}
+
+const ConferenceCard = (props: Readonly<ConferenceCardProps>) => {
+  const [talk, setTalk] = useState<TalkStateProps>();
+  const [speakers, setSpeakers] = useState<SpeakerStateProps[]>([]);
+
+  if (!props.activities.slug) {
+    return;
+  }
+
+  const fetchTalk = async () => {
+    try {
+      if (props?.activities?.slug) {
+        const fetchedTalk = await getEntry("talks", props?.activities?.slug);
+        setTalk(fetchedTalk as TalkStateProps);
+
+        if (fetchedTalk?.data?.speakers) {
+          const fetchedSpeakers = await Promise.all(
+            fetchedTalk?.data?.speakers.map(async (speaker: any) => {
+              return await getEntry("speakers", speaker);
+            }),
+          );
+          setSpeakers(fetchedSpeakers);
+        }
+      }
+    } catch (error) {
+      console.error("Failed to fetch talk data:", error);
+    }
+  };
+
+  useEffect(() => {
+    fetchTalk();
+  }, [props.activities.slug]);
+
+  return (
+    <div className="flex flex-col gap-4">
+      <a
+        href="/"
+        className={cn(
+          "flex w-full flex-[4] gap-2 rounded-lg border-2 border-neutral-600 bg-neutral-900 p-4",
+          props.activities.type === "break" && "bg-neutral-800",
+        )}
+      >
+        <div className="flex w-full">
+          <div className="flex w-full flex-col gap-4">
+            <div className="flex flex-row justify-between gap-4">
+              <div className="flex flex-col gap-1">
+                <p className="font-heading text-lg font-medium leading-6">
+                  {talk?.data?.title || "Conference Name"}
+                </p>
+                {talk?.data?.speakers && (
+                  <div className="text-sm font-semibold text-gray-300">
+                    {talk?.data?.speakers.map((speakerSlug, index) => (
+                      <p key={index}>
+                        {speakers?.find(
+                          (speaker) => speaker.slug === speakerSlug,
+                        )?.data?.name || "Unknown Speaker"}
+                      </p>
+                    ))}
+                  </div>
+                )}
+              </div>
+              {(props.activities.type === "conference" ||
+                props.activities.type === "roundtable") &&
+                speakers.length > 0 && (
+                  <div className="grid grid-cols-2 gap-2">
+                    {talk?.data?.speakers.map(
+                      (speakerSlug: unknown, index: number) => {
+                        const speaker = speakers.find(
+                          (s) => s.slug === speakerSlug,
+                        );
+                        return speaker ? (
+                          <div
+                            key={speaker.id || index}
```

Comment:
i dont think we will ever see a conference without speakers , so 
```suggestion
                            key={speaker.id}
```